docs/news/index.qmd

```diff
@@ -9,6 +9,95 @@ format:
+- Added support for accessing delegated credentials through Azure Active Directory in RStudio Pro sessions (rstudio/rstudio-pro#4485)
+- Support for setting per-user AWS credentials in RStudio Pro sessions is now generally available (rstudio/rstudio-pro#4246)
+
+### Fixed
+
+#### RStudio IDE
+- Fixed display problems with Choose R dialog when UI language is French (#12717)
+- Fixed focus switching to Help Pane search box after executing ? in the console [Accessibility] (#12741)
+- Fixed initial focus placement in Help Pane [Accessibility] (#10600)
+- Fixed invalid element role on session-suspended icon [Accessibility] (#12449)
+- Improve screen-reader support for Console pane toolbar [Accessibility] (#12825)
+- Background script jobs are now run using the global environment. This fixes the behaviour of `source()` in backgrounds jobs. (#11866)
+- Fixed bug that caused Update Available dialog to show after the user selected to Ignore Update (rstudio/rstudio-pro#4179)
+- Fixed bug that prevented updating to the latest release if version was previously ignored (#12874)
+- RStudio no longer uses `reg.exe` when attempting to enumerate R versions in the Windows registry (#12599)
+- Fixed file-type icons not displaying in Finder on Mac (#12252)
+- Fixed saving and restoring window location when maximized or partially offscreen (#12463)
+- Fixed display of macOS message dialogs (#12928)
+- Set theme of menu bar, title bar, and dialogs (dark vs. light) based on RStudio theme (#12247)
+- Fixed issues with mouse back / forward navigation in Source pane, Help pane (#12932)
+- Fixed opening files from command-line with relative paths (#12495, #12563)
+- Fixed issue with column preview with older versions of the 'pillar' package. (#12863)
+- Fixed bug where the OK button was disabled in Choose R dialog when only one version of R installed (#12916)
+- Improved robustness when chosing custom R version in Windows desktop (#12969)
+- Fixed bug that prevented RStudio Desktop from starting on Linux if desktop.ini was unreadable (#12963)
+- Improved RStudio Desktop startup behavior when user state folder inaccessible (#12988)
+- Fixed saving files to UNC paths on Windows (#12652, #12935)
+- Fixed window disappearing when restoring from maximized on macOS (#13010)
+- Fixed Project Build Tools preferences pane showing &mdash text in French UI (#11134)
+- Fixed inserting cross references in a Quarto document showing no references (#12882)
+- Fixed keyboard shortcut help not displaying in Emacs or Sublime Text mode (#11376)
+- Fixed issue with rendering development documentation with R 4.3.0 (#12945)
+- Fixed issue with use of "Whole word" with Find in Files search (#13017)
+- Fixed issue with diagnostics freezing session with documents containing pipebind operator (#13091)
+- Fixed issue with stopping a Quarto Shiny doc render on Windows (#12546)
+- Fixed issue causing sessions to fail to start if repos contained NA value (#12946)
```

Comment:
The next entry has NA as `NA` values - should these be the same or are they ok as is?

Reply:
Probably best to have it be consistent.